spec/components/provider_interface/interview_form_component_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+
+RSpec.describe ProviderInterface::InterviewFormComponent do
+  let(:interview_preferences) { nil }
+  let(:application_form) { build_stubbed(:application_form, interview_preferences: interview_preferences) }
+  let(:application_choice) { build_stubbed(:submitted_application_choice, application_form: application_form) }
+
+  let(:form_object_class) do
```

Comment:
This makes me think the component and wizard are too tightly coupled. I experimented with moving the provider list logic into the component, but really the wizard needs to know about this too. Any suggestions for a better place for this?

Reply:
Hmm good question, maybe if we move the logic for `application_providers` and `multiple_application_providers` into the `application_choice` as it does feel like its that model's concern, then we just use it straight in the view via `application_choice`, what do you think?

Reply:
That sounds good! 👍